core-blocks/embed/style.scss

```diff
@@ -6,3 +6,12 @@
 	max-width: $content-width / 2;
 	width: 100%;
 }
+
+.wp-block-embed {
+	// Supply caption styles to images, even if the theme hasn't opted in.
+	// Reason being: the new markup, figcaptions, are not likely to be styled in the majority of existing themes,
```

Comment:
Is this intended to be deprecated eventually, once themes know about this new markup? Or will we keep it?

This comment reads a bit like it's a stop-gap.

Reply:
I think we should phase it away, but don't want to rush it because it will cause a visual degradation at this moment.